deploy-manage/monitor/autoops/autoops-sm-troubleshoot-custom-certification.md

```diff
@@ -0,0 +1,118 @@
+---
+applies_to:
+  deployment:
+    self:
+    ece:
+    eck:
+navigation_title: Elastic agent not recognizing SSL certificate
+products:
+  - id: cloud-kubernetes
+  - id: cloud-enterprise
+---
+
+# Troubleshoot {{agent}} not recognizing custom SSL certificate 
+
+If {{agent}} is failing to connect your self-managed cluster to AutoOps because it doesn't recognize your SSL certificate, this might be because your certificate is signed by a custom or internal Certificate Authority (CA). 
+
+You might encounter an error like the following:
+
+```sh
+... x509: certificate signed by unknown authority ...
+```
+
+This error occurs because the machine where you have installed {{agent}} does not trust your custom or internal CA. To fix this error, you need to configure the agent with your custom certificate by providing its certificate file so that the machine trusts your CA.
+
+This following section provides the steps to configure {{agent}} with a custom SSL certificate so that the agent can successfully connect your self-managed cluster to AutoOps.
+
+## Configure AutoOps {{agent}} with a custom SSL certificate
+
+To configure {{agent}} with your custom SSL certificate, you must edit the `elastic-agent.yml` file on the agent's host machine to add the path to your certificate. The host machine is the machine where you have installed the agent. 
+
+Complete the following steps:
+
+1. On the host machine, open the `elastic-agent.yml` file. \
+  The default location is `/opt/Elastic/Agent/elastic-agent.yml`.
```

Comment:
```suggestion
1. On the host machine, open the `elastic-agent.yml` file.
  The default location is `/opt/Elastic/Agent/elastic-agent.yml`.
```

Personally, I'd keep these two statements on one line, but maybe there's some style guideline that I'm not aware of. :-)